src/app-layout/__integ__/app-layout-tools.test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../lib/components/test-utils/selectors';
+import { viewports } from './constants';
+
+const wrapper = createWrapper().findAppLayout();
+class AppLayoutToolsViewPage extends BasePageObject {
+  async openTools() {
+    await this.click(wrapper.findToolsToggle().toSelector());
+  }
+
+  async closeTools() {
+    await this.click(wrapper.findToolsClose().toSelector());
+  }
+}
+
+describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme => {
+  function setupTest(testFn: (page: AppLayoutToolsViewPage) => Promise<void>, path: string) {
+    return useBrowser(async browser => {
```

Comment:
You can put viewport size here as an extra argument and save one resize operation

```suggestion
    return useBrowser({ ...viewports.desktop }, async browser => {
```